src/ui/ResultList/ResultList.ts

```diff
@@ -331,6 +360,15 @@ export class ResultList extends Component {
     return $$(this.options.resultContainer).findAll('.CoveoResult');
   }
 
+  public enable() {
+    super.enable();
+    this.element.style.display = 'block';
```

Comment:
We had some problems with elements using `style.display = 'block'` since it basically overrides everything.

In my experience, toggling a class `.hide` is better since it can be easily overridden in CSS by defining a rule with higher priority: It becomes a CSS-resolved problem instead of a JavaScript one. 

Plus, it is way easier to revert to the earlier state, since you are simply "not applying" a `display:none` style anymore instead of hardcoding a display style.